plugins/plugin-chart-table/src/controlPanel.tsx

```diff
@@ -192,7 +192,39 @@ const config: ControlPanelConfig = {
             },
           },
         ],
-        ['row_limit'],
+        [
+          {
+            name: 'backend_pagination',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Backend pagination'),
+              description: t('Enable backend pagination'),
+              default: false,
+            },
```

Comment:
The label should probably be "Server pagination" and the description "Enable server side pagination of results"